README.md

```diff
@@ -58,7 +73,32 @@ You should always shut down `HTTPClient` instances you created using `try httpCl
 
 ## Usage guide
 
-Most common HTTP methods are supported out of the box. In case you need to have more control over the method, or you want to add headers or body, use the `HTTPRequest` struct:
+The default HTTP Method is `GET`. In case you need to have more control over the method, or you want to add headers or body, use the `HTTPClientRequest` struct:
+#### Using Swift Concurrency
+```swift
```

Comment:
As a mild nit, a bit more whitespace around this would make it read nicer when we look at it in plaintext.